Misc/NEWS.d/next/Library/2017-12-15-15-34-12.bpo-32248.zmO8G2.rst

```diff
@@ -1,7 +1,8 @@
 Add :class:`importlib.abc.ResourceReader` as an ABC to provide a
 unified API for reading resources contained within packages. Loaders
 wishing to support resource reading are expected to implement the
-``get_resource_reader(fullname)`` method.
+``get_resource_reader(fullname)`` method.  File-based and zipimport-based
+loaders both implement these APIs.
 
 Also add :mod:`importlib.resources` as the stdlib port of the
 ``importlib_resources`` PyPI package. The modules provides a high-level
```

Comment:
This is a very long NEWS entry that looks like it should probably be in What's New :)

Reply:
I think you're right :).  I'll shorten the NEWS entry and expand a bit in the What's New entry.